_notices/rsn0022.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 22 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Deprecation announcement for Python v3.9 in v22.12"
+notice_author: RAPIDS TPM
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v22.12+"
+notice_created: 2022-11-21
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-11-21
+---
+
+## Overview
+
+Python 3.9 will be deprecated in our upcoming release of v22.12 scheduled for December 2022. After the v22.12 release, users should expect that support for Python 3.9 may be dropped at any point.
+
+
+## Impact
+
+Users should plan to move to Python 3.10 as soon as possible. Future support of additional Python 3.11 version is being worked on for upcoming releases. Further details will be shared in RSN notices as they become available.

```

Comment:
@aravenel I wondered the same. Would it be less confusing if we combine the two RSNs into one rather than cross-linking each to the other? A single RSN for “Drop Python 3.9, Add Python 3.10” and a single RSN for “Drop Ubuntu 18.04, Add Ubuntu 22.04.”

Reply:
I agree in this case. Historically we have separated them as they haven't always been so tightly coupled, but feels like we tying ourselves in knots here for no reason. @caryr35 what do you think?